### PR TITLE
add short summary to README

### PR DESCRIPTION
**Pull request by Zsailer**
Addressing #38. 

@consideRatio how does this look? 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+[![CircleCI](https://circleci.com/gh/jupyter/telemetry.svg?style=svg)](https://circleci.com/gh/jupyter/telemetry)
 [![codecov](https://codecov.io/gh/jupyter/telemetry/branch/master/graph/badge.svg)](https://codecov.io/gh/jupyter/telemetry)
 [![Documentation Status](https://readthedocs.org/projects/jupyter-telemetry/badge/?version=latest)](https://jupyter-telemetry.readthedocs.io/en/latest/?badge=latest)
 
-Telemetry for Jupyter Applications and extensions.
+*Telemetry for Jupyter Applications and extensions.*
 
+> Telemetry (təˈlemətrē): the process of recording and transmitting the readings of an instrument. [Oxford Dictionaries]
+
+Jupyter Telemetry enables Jupyter Applications (e.g. Jupyter Server, Jupyter Notebook, JupyterLab, JupyterHub, etc.) to record **events**—i.e. actions by application users—and transmit them to remote (or local) logs as **structured** data. It works with Python's standard `logging` library to handle the transmission of events allowing users to send events to local files, over the web, etc.
```
Comment (consideRatio) [inline README.md]: "to remote or local logs" - I stopped to think about this part. Does this phrasing it suggest a limit of what you do when receiving these events from jupyter_telemetry to logging specifically?

Comment (consideRatio) [reply README.md]: Destinations?

Comment (Zsailer) [reply README.md]: Great points! I think remote or local "destinations" is a much better phrasing.

**Comment by Zsailer** [comment]
Thanks, @consideRatio!

**Comment by consideRatio** [comment]
Thank you @Zsailer ! :tada:

**Pull request merged**